compute_tools/src/http/routes/prewarm_lfc.rs

```diff
@@ -0,0 +1,49 @@
+use crate::compute::{ComputeNode, EndpointStoragePair, PrewarmStatus};
+use axum::response::{IntoResponse, Response};
+use axum::{Json, http::StatusCode};
+use compute_api::responses::PrewarmOffloadState;
+use std::sync::Arc;
+use tracing::error;
+
+type State = axum::extract::State<Arc<ComputeNode>>;
+
+impl axum::extract::FromRequestParts<Arc<ComputeNode>> for EndpointStoragePair {
+    type Rejection = Response;
+    async fn from_request_parts(
+        _: &mut http::request::Parts,
+        state: &Arc<ComputeNode>,
```

Comment:
NIT, but it impacts readability a lot -- `state.state` -- what state of what state? I suggest making it more clear

```suggestion
        compute: &Arc<ComputeNode>,
```